Firestore/core/src/firebase/firestore/util/config.h

```diff
@@ -0,0 +1,33 @@
+/*
```

Comment:
This was the old location of the file (before we shortened all the paths). This should be removed.